src/gui/editors/AutomationEditor.cpp

```diff
@@ -862,6 +925,56 @@ void AutomationEditor::mouseMoveEvent(QMouseEvent * mouseEvent )
 				}
 			break;
 			}
+			case EditMode::EditTangents:
```

Comment:
It's close in size to the `EditMode::Draw` case I think. If you don't mind I'd rather leave this for a PR that would rework all the cases, since most are large.